src/core_plugins/tag_cloud/public/arcinput_directive.js

```diff
@@ -26,7 +26,12 @@ module.directive('arcInput', function ($timeout) {
 
       scope.$on('$destroy', () => {
         $timeout.cancel(resizeTimeout);
-        element[0].firstChild.children[0].removeEventListener(onOrientationChange);
+        try {//gross, but watill be deletd anyway
```

Comment:
sorry, that file needs to be deleted. it was a mistake to check this in, let alone the garbled language